docs/serving/samples/knative-routing-go/sample-internal.yaml

```diff
@@ -0,0 +1,57 @@
+# Copyright 2018 The Knative Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: serving.knative.dev/v1
+kind: Service
+metadata:
+  name: search-service
+  namespace: default
+  annotations:
```

Comment:
I think visibility setting is not `annotations` but `labels` - https://knative.dev/docs/serving/cluster-local-route/

Reply:
Absolutely!